Assignment_Folder/saumya.txt

```diff
@@ -1,2 +1 @@
-Hi, I am Saumya 
-
+Hi I'm Richika
```

Comment:
Hi @RICHIKA-RANA !

Glad to see your PR :)

Could you please make a separate file `Richika.txt` and add your bio there? Also please leave the file `saumya.txt` unedited i.e remove the extra line you added :)

Thanks!